shared/types/Tournament/index.d.ts

```diff
@@ -26,7 +26,7 @@ export interface Match {
 export default interface Tournament {
   id?: string;
   name: string;
-  ownerMSId: string;
+  owner: User;
```

Comment:
Comparing to our main branch and every CRUD functions that we currently implement, it should stay like it is on main branch.

Reply:
I prepared this branch to work with resolving userId by backend. I think this change should happen in a few days